imports/api/collections/students.js

```diff
@@ -261,7 +261,12 @@ Profile = new SimpleSchema({
     type: [StudentsSchema5],
     label: "Experiência Profissional",
     optional: true
-  }
+  },
```

Comment:
And same as above, but in here I'm keeping all job applications for the student, we are simulating **primary** and **foreign keys**